src/Utils/hiopOptions.cpp

```diff
@@ -1092,32 +1092,43 @@ void hiopOptionsPriDec::register_options()
   // convergence and stopping criteria
   //
   {
+    register_num_option("alpha_max",
+                        1e6,
+                        1e14,
+                        1,
+                        "Upper bound of quadratic coefficient alpha (default 1e6)");
```

Comment:
function signature of register_num_option is `void register_num_option(const std::string& name, double defaultValue, double rangeLo, double rangeUp, const char* descr);`
The 3rd input argument should be the lower bound and the 4th argument should be the upper bound. 
I think you just misuse them in all the new options you introduced.

Reply:
You're correct. Very good catch.